### PR TITLE
CHANGE: Added Windows Input tests to ensure Mouse movement works usin…

### PR DESCRIPTION
**Pull request by karljj1**
Added a test to go with the fix in https://ono.unity3d.com/unity/unity/pull-request/102187/_/se/input/rdp-mouse-position-fix

CHANGE: Added Windows Input tests to ensure Mouse movement works using RDP.

### CODE REVIEW

**Comment by karljj1** [comment]
The tests are expected to fail until the fix lands

File: Assets/Tests/InputSystem/WindowsTests.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+using UnityEngine.InputSystem;
+using UnityEngine.TestTools;
+
+partial class WindowsTests
+{
+    [UnityTest]
+    [Category("WindowsInput")]
+    public IEnumerator WindowsInput_RemoteDesktopMouseMovements_AreDetected()
+    {
+        var mouse = InputSystem.GetDevice<Mouse>();
```
Comment (Rene-Damm) [inline Assets/Tests/InputSystem/WindowsTests.cs]: `Mouse.current`

**Review by Rene-Damm — COMMENTED** [review]
Since this goes end-to-end, would prefer to have this as part of `IntegrationTests`.

Also, I assume this can't land until we've landed the fix on the native side?

File: Assets/Tests/InputSystem/WindowsTests.cs (hunk 1 above)
Comment (karljj1) [reply Assets/Tests/InputSystem/WindowsTests.cs]: You want it moved into the IntegrationTests folder?

**Comment by karljj1** [comment]
Wont be able to backport this fix further than 2019.3 so these tests wont pass. Ill have a think about a solution, maybe move the test into trunk.

**Pull request merged**